configuration/driver/icedrv_history.F90

```diff
@@ -364,17 +431,17 @@ subroutine history_write()
       start4(1) = 1
       count4(1) = nx
       start4(2) = 1
-      count4(2) = ncat
+      count4(2) = ntrcr
       start4(3) = 1
-      count4(3) = ntrcr
+      count4(3) = ncat
```

Comment:
does this change only impact FSD output?

Reply:
It impacts all the variables in trcrn, which includes many other tracers. All of them are currently set to -9999 in the history file, even without the ITD.